src/core/server/core_app/integration_tests/static_assets.test.ts

```diff
@@ -23,7 +23,7 @@ describe('Platform assets', function () {
   let root: Root;
 
   beforeAll(async function () {
-    root = kbnTestServer.createRoot();
+    root = kbnTestServer.createRoot({ plugins: { initialize: false } });
```

Comment:
Just to understand, why did you have to add the `{ plugins: { initialize: false } }` option to these core integration tests? Is that just test perf optimization?

Reply:
@joshdover helped me with that to fix these type of issues: https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/69612/

Reply:
Oh right, that is adding an ES call during `start` so it would cause ES connection errors. Thanks.